components/well-bold/well-bold-en.html

```diff
@@ -0,0 +1,75 @@
+---
+breadcrumb: [
+	{ "title": "GCWeb home", "link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html" },
+	{ "title": "Provisional functionality", "link": "https://wet-boew.github.io/themes-dist/GCWeb/provisional-en.html" }
+]
+dateModified: "2021-05-25"
+description: "Documentation and working example for the well bold - provisional CSS"
+language: "en"
+altLangPage: "well-bold-fr.html"
+tag: "well-bold"
+title: "Well bold - Provisional CSS"
+---
+{{>alert-provisional-demos}}
+<span class="wb-prettify all-pre hide"></span>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Provisional</dd>
+	<dt>Type</dt>
+	<dd>Utility</dd>
+	<dt>Last review</dt>
+	<dd>2021-05-25</dd>
+	<dt>Conforming to</dt>
+	<dd>n/a</dd>
+	<dt>Nom technique</dt>
+	<dd><code>well-bold</code></dd>
+</dl>
+
+<h2>Purpose</h2>
+<p>This apply font weight bold to evey text element inside a well component and generates normal font weight when the element <code>strong</code> is applied.</p>
+
+<h2>Working example</h2>
+
+<h3>Well bold (provisional v1.0)</h3>
+<section class="well well-sm provisional well-bold brdr-0 mrgn-bttm-0">
```

Comment:
Please demo only the well-bold on it's own. Remove the other feature unrelated CSS class